github_api/prs.py

```diff
@@ -105,6 +107,16 @@ def get_pr_comments(api, urn, pr_num):
     for comment in comments:
         yield comment
 
+def owner_name_added_in_diff(api, urn, pr_num, owner_name):
+    """
+    Attempt to match the owner name in a diff as an addition.
+    Returns True if the owner_name is in the diff on an addition line.
+    """
+    path = 'http://github.com/{urn}/pull/{pr_num}.diff'
+    diff = api('get', path)
+    regex = '\+.+{username}'.format(username=owner_name)
```

Comment:
This requires there be at least one character between the literal '+' and the username. You probably one `.*` instead of `.+`. It'd also be slightly more correct if you used `^` with MULTILINE.